examples/pytorch/gcmc/model.py

```diff
@@ -336,12 +338,9 @@ def forward(self, graph, ufeat, ifeat):
             ufeat = self.dropout(ufeat)
             ifeat = self.dropout(ifeat)
             graph.nodes['movie'].data['h'] = ifeat
-            basis_out = []
-            for i in range(self._num_basis):
-                graph.nodes['user'].data['h'] = ufeat @ self.Ps[i]
-                graph.apply_edges(fn.u_dot_v('h', 'h', 'sr'))
-                basis_out.append(graph.edata['sr'].unsqueeze(1))
```

Comment:
Note that under new kernel there is no need for a `unsqueeze` operation here. Please checkout the code on master branch.